data/tutorials/gs_00_up_and_running.md

```diff
@@ -115,21 +115,21 @@ Now check the installation by running `opam --version`. You can compare it with
 
 #### Creating a New Switch on Unix
 
-If you want a specific version of OCaml, or a new independent environment, you can create a new switch with the `opam switch create` command. Specify which version as shown below (i.e., `opam switch create 4.14.0`). All possible compiler versions can be found with `opam switch list-available`. The most current version can be found at [opam.ocaml.org](https://opam.ocaml.org/packages/ocaml-base-compiler/).
+If you want a specific version of OCaml, or a new independent environment, you can create a new switch with the `opam switch create` command. Specify which version as shown below (i.e., `opam switch create 5.0.0~beta0`). All possible compiler versions can be found with `opam switch list-available`. The most current version can be found at [opam.ocaml.org](https://opam.ocaml.org/packages/ocaml-base-compiler/).
```

Comment:
Agree wholeheartedly with @avsm's suggestion (in general, `opam switch create <version>` is a hangover-hack from OPAM 1.x days; it's better in docs and examples always to specify `ocaml.<version>` since `opam switch create 5.0.0` will fail if you've installed OCaml 5.0.0 via your OS package manager.
```suggestion
If you want a specific version of OCaml, or a new independent environment, you can create a new switch with the `opam switch create` command. Specify which version as shown below (i.e., `opam switch create ocaml-5.0 ocaml.5.0.0`). All possible compiler versions can be found with `opam switch list-available`. The most current version can be found at [opam.ocaml.org](https://opam.ocaml.org/packages/ocaml-base-compiler/).
```
(same edits then on all the hunks below, obviously)